app/models/chargeback_rate_detail.rb

```diff
@@ -225,6 +229,10 @@ def contiguous_tiers?
 
   private
 
+  def gratis?
+    chargeback_tiers.all?(&:gratis?)
```

Comment:
just note: when first tier is gratis, there are no other tiers.
So `chargeback_tiers.first.gratis?` should be is enough